plugins/blanker_spectrum.py

```diff
@@ -65,6 +77,68 @@ def _unlinkHwAxes(self):
         except Exception:
             logging.exception("Failed to set the blanker tp automatic mode")
 
+    def onBackgrounFile(self, fn):
```

Comment:
Seems a spelling mistake onBackgrounFile in stead of onBackgroun**d**File

Reply:
scharp!